content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -1,130 +1,129 @@
-=========================================================
-Get correct tax rates in the United States using TaxCloud
-=========================================================
-
-The **TaxCloud** integration allows you to correctly calculate the sales
-tax for every address in the United States and keeps track of which products
-are exempt from sales tax and in which states each exemption applies.
-TaxCloud calculates sales tax in real-time for every state, city, and
+========================
+Tax rates using TaxCloud
+========================
+
+The **TaxCloud** integration allows you to correctly calculate the sales tax for every address in
+the United States and keeps track of which products are exempt from sales tax and in which states
+each exemption applies. TaxCloud calculates sales tax in real-time for every state, city, and
 special jurisdiction in the United States.
 
 Configuration
 =============
 
 In TaxCloud
 -----------
-* Create a free account on `TaxCloud
-  <https://taxcloud.com/#register>`__ website.
-* Register your Odoo website on TaxCloud to get an *API ID* and an *API Key*.
+#. Create a free account on `TaxCloud <https://taxcloud.com/#register>`__ website.
+#. Register your Odoo website on TaxCloud to get an **API ID** and an **API Key**.
+#. In :guilabel:`Settings` on TaxCloud, click :guilabel:`Locations` to enter the location of your
+   Office(s) and Warehouse(s).
+#. In :guilabel:`Settings` on TaxCloud, click :guilabel:`Manage Tax States` to verify the states
+   in which you collect sales tax.
 
 .. image:: taxcloud/taxcloud01.png
-  :align: center
-
-* In Settings on TaxCloud, click *Locations* to enter the location of your Office(s) & Warehouse(s).
-* In Settings on TaxCloud, click *Manage Tax States* to verify the states where you collect sales tax.
+   :align: center
+   :alt: TaxCloud account creation.
```

Comment:
Should it be resized? The image is very big

Reply:
Fixed